timeseries/tests/unittests/models/test_models.py

```diff
@@ -28,7 +28,7 @@
     get_data_frame_with_item_index,
     to_supported_pandas_freq,
 )
```

Comment:
Should we run platform tests for a good measure?

Reply:
platform tests look good at https://github.com/autogluon/autogluon/actions/runs/11781956077